src/analyze-action-cleanup.ts

```diff
@@ -0,0 +1,34 @@
+import * as core from "@actions/core";
+
+import * as actionsUtil from "./actions-util";
+import { Config, getConfig } from "./config-utils";
+import { getActionsLogger } from "./logging";
+
+async function run(uploadSarifDebugArtifact: Function) {
+  const logger = getActionsLogger();
+
+  let config: Config | undefined = undefined;
+  config = await getConfig(actionsUtil.getTemporaryDirectory(), logger);
```

Comment:
This should be enough, since you assign the variable right away.
```suggestion
  const config = await getConfig(actionsUtil.getTemporaryDirectory(), logger);
```

Reply:
Makes sense, thanks!